config/filepond.php

```diff
@@ -13,12 +13,15 @@
 
     /*
     |--------------------------------------------------------------------------
-    | Local Temporary Path
+    | Temporary Path
     |--------------------------------------------------------------------------
     |
     | When initially uploading the files we store them in this path
+    | By default, it is stored on the local disk which defaults to `/storage/app/{temporary_files_path}`
     |
     */
-    'temporary_files_path' => realpath(sys_get_temp_dir()),
+    'temporary_files_path' => env('FILEPOND_TEMP_PATH', 'filepond'),
```

Comment:
The problem I ran into was that the system temp dir wasn't accessible by Laravel's disk system unless it was set up in the config file. Adding an extra step for the temp disk didn't make sense to me for the 99% use-case.

We can look into adding it via the service provider if you want to keep using the system temp dir, otherwise, I would suggest following along other packages that use Laravel's storage directory for temp storage (like Spatie's Media Library package). If you think it will be a problem, you could do a major release since it is a breaking change.

Reply:
Hmm yeah.. As we are still <v1 a major release isn't really a possibility, but also, as we are still <v1 having such a small breaking change is probably fine.